gradle/proguard-rules.pro

```diff
@@ -74,8 +74,8 @@
 -dontwarn java.time.zone.ZoneRulesProvider
 
 # DataBinding
--keep public class * extends androidx.databinding.ViewDataBinding {
-  * inflate(android.view.LayoutInflater);
+-keep,allowoptimization public class * extends androidx.databinding.ViewDataBinding {
+  public static * inflate(android.view.LayoutInflater);
```

Comment:
This dex file defines 3858 classes with 19339 methods, and references 24599 methods